lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reuse-existing-identity.yaml

```diff
@@ -138,6 +138,8 @@ states:
         targetState: UPDATE_NAME_DOB_PAGE
       cancel:
         targetState: RETURN_TO_RP
+      back:
+        targetState: IDENTITY_REUSE_PAGE_COI
```

Comment:
I think a few states can lead to the `UPDATE_DETAILS_PAGE` state

Reply:
Bah. Fix incoming